src/common/readout/ess/Parser.h

```diff
@@ -56,6 +56,7 @@ class Parser {
     NMX = 0x44,
     FREIA = 0x48,
     ESTIA = 0x4C,
+    AMOR = 0x4E,
```

Comment:
I think we should keep AMOR out of this despite what I might have said earlier :-)

At ESS we will use TBL VMM (as specified in the Readout ICD). The DetectorType 
enums should be checked. I can see CSPEC is wrong andTREX should be added. Also TBLVMM
shoulkd be added.